packages/voting-module-adapter/adapters/cw4-voting/actions/makeManageMembersAction/ManageMembersComponent.tsx

```diff
@@ -76,10 +76,10 @@ export const ManageMembersComponent: ActionComponent<ManageMembersOptions> = ({
                     disabled={!isCreating}
                     error={errors?.toAdd?.[index]?.weight}
                     fieldName={weightFieldName}
-                    onPlusMinus={[
-                      () => setValue(weightFieldName, Math.max(weight - 1, 0)),
-                      () => setValue(weightFieldName, weight + 1),
-                    ]}
+                    onMinus={() =>
```

Comment:
I think you mixed up the plus and minus functions here. The plus should be first and the minus second in `onPlusMinus`.

Reply:
Great catch! This was the motivation for making this change. The changes should fix the issue, since the incorrect `onPlusMinus` array is removed here.

Reply:
I see. The `onPlusMinus` is incorrect.